.github/workflows/python.yml

```diff
@@ -0,0 +1,47 @@
+name: Python
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.12']
```

Comment:
The current Python version is good for now. :+1:

Reply:
Actually, this list maybe should match with project classifiers in `pyproject.toml`. Currently there's a list of supported versions from `3.8` up to `3.13`. I would drop most of them out from there now.